test/storage-memory.cloudflare.test.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+const assert = require('node:assert')
+const { after, before, describe, test } = require('node:test')
+const { unstable_dev } = require('wrangler') // eslint-disable-line camelcase
+const { promisify } = require('util')
+
+const sleep = promisify(setTimeout)
+
+describe('storage memory (cloudflare)', async () => {
```

Comment:
can you please skip this in Node.js v18.6?

Reply:
Skipped in c774406

Reply:
Looks like it's still not skipping the suite in 18 - perhaps `before` is still being run for the suite? I'm sorry, this is my first experience with `node:test`, and I'm not entirely certain how to solve this

Reply:
Although it appears to be documented for v18, I couldn't get the option to work when running 18.19.0 locally. I just put the whole `describe` in an `if` block in d1a4ed7, and ran it locally against v18.19.0